### ISSUE TITLE
wrf_chem_mm reader loading two different things as height

### ISSUE THREAD

**Issue by blychs** [noaa-oar-arl/monetio#194]
@zmoon @mlirenzhenmayi 
The _wrfchem_mm.py reader seems to be loading a variable called "height" twice (when var is 'height' and when var is 'zstag'). This causes xr.merge() to fail, at least in my system.
I think that the issue is that, in the version I installed (with pip after cloning the main branch on github, so it should be the latest), zstag is also called "height", although it has bottom_top_stag as vertical dimension.
Easy solution: adding a rename for zstag:
```
  101         elif var in {"height", "height_agl", "zstag"}:                                  
  102             var_wrf = getvar(                                                           
  103                 wrflist, var, timeidx=ALL_TIMES, method="cat", squeeze=False, units="m" 
  104             )
```
to:
```
  101         elif var in {"height", "height_agl", "zstag"}:                                  
  102             var_wrf = getvar(                                                           
  103                 wrflist, var, timeidx=ALL_TIMES, method="cat", squeeze=False, units="m" 
  104             )                                                                           
  105             if var == "zstag":                                                          
  106                 var_wrf = var_wrf.rename("zstag")  
```

This should work also with older versions.
Cheers
Pablo

**Comment by zmoon** [comment]
Yep I can reproduce that.

**Comment by blychs** [comment]
Solved in #195
